README.md

```diff
@@ -101,6 +101,13 @@ Other options for `install`:
 * `--k3s-extra-args` - Optional extra arguments to pass to k3s installer, wrapped in quotes, i.e. `--k3s-extra-args '--no-deploy traefik'` or `--k3s-extra-args '--docker'`. For multiple args combine then within single quotes `--k3s-extra-args '--no-deploy traefik --docker'`.
 * `--k3s-version` - set the specific version of k3s, i.e. `v0.9.1`
 - `--ipsec` - Enforces the optional extra argument for k3s: `--flannel-backend` option: `ipsec`
+* `--ha-mysql` - Use an external MySQL datastore for HA
+* `--ha-psql` - Use an external PostgreSQL database for HA
+* `--db-hostname` - Hostname of external database server (Used with either `--ha-mysql` or `--ha-psql`)
+* `--db-port` - Port where the external database server is listening on (Used with either `--ha-mysql` or `--ha-psql`)
+* `--db-username` - Username of external database server (Used with either `--ha-mysql` or `--ha-psql`)
+* `--db-passwd` - Password of external database user (Used with either `--ha-mysql` or `--ha-psql`)
+* `--db-name` - Database to use for k3s (Used with either `--ha-mysql` or `--ha-psql`)
 * See even more install options by running `k3sup install --help`.
 
```

Comment:
Please add a heading showing an example of how to configure postgresql (all the flags needed)